### PR TITLE
Feature/school 712/cookie block

### PR DESCRIPTION
**Pull request by levil664**

### CODE REVIEW

File: apps/schools/domains/user/components/auth/containers/Cookies.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import Image from 'next/image'
+import arrowBottom from '@public/icons/arrowBottom.svg'
+import arrowTop from '@public/icons/arrowTop.svg'
+import { Button } from '@domains/common/components/button'
+import { Typography } from 'antd'
+import Cookie from 'universal-cookie'
+
+const COOKIE_AGREEMENT_KEY = 'cookieAgreementAccepted'
+
+export const Cookies = () => {
```
Comment (mzavarka) [inline apps/schools/domains/user/components/auth/containers/Cookies.tsx]: maybe rename it?
CookieComponent or PopupCookie

File: apps/schools/domains/user/components/auth/containers/Cookies.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    }, [])
+
+    useEffect(() => {
+        const cookieValue = cookies.get(COOKIE_AGREEMENT_KEY)
+        if (cookieValue) {
+            setCookiesNotAccepted(false)
+        }
+    }, [])
+
+    if (!cookiesNotAccepted) {
```
Comment (mzavarka) [inline apps/schools/domains/user/components/auth/containers/Cookies.tsx]: did you find some references what we should do if user refuse to use cookies?
should we check everywhere using cookie this variable?

Comment (levil664) [reply apps/schools/domains/user/components/auth/containers/Cookies.tsx]: I do not understand at all why these popups are with the word "accept" (the policy is meant) anyway cookies are used

Comment (levil664) [reply apps/schools/domains/user/components/auth/containers/Cookies.tsx]: The doma.ai have this popup, but they always use cookies

**Pull request merged**